src/components/coin-divider/CoinDividerOpenButton.js

```diff
@@ -31,77 +25,71 @@ const CaretIcon = styled(ImgixImage).attrs(({ theme: { colors } }) => ({
   width: 8;
 `;
 
-const ContainerButton = styled(ButtonPressAnimation).attrs(
-  ({ isSmallBalancesOpen, isSendSheet }) => ({
-    scaleTo: 0.9,
-    wrapperStyle: {
-      marginLeft: isSendSheet && android ? 16 : 0,
-      width: isSmallBalancesOpen ? 80 - (android ? 4 : 0) : closedWidth - 4,
-    },
-  })
-)`
-  width: ${({ isSmallBalancesOpen }) =>
-    isSmallBalancesOpen ? 80 : closedWidth};
-`;
-
-const Content = styled(Row).attrs({
+const Content = styled(Animated.View).attrs({
   align: 'center',
+  flexDirection: 'row',
   justify: 'space-between',
 })`
+  background-color: ${({ theme: { colors } }) => colors.blueGreyDarkLight};
   ${padding(0, 10)};
   border-radius: ${RoundButtonCapSize / 2};
   height: ${({ height }) => height};
-  width: ${closedWidth};
 `;
 
+const LabelText = styled(Text).attrs(({ shareButton, theme: { colors } }) => ({
```

Comment:
The `shareButton` prop doesn't appear to be used anywhere.

Reply:
Also fixed this